ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -432,6 +443,8 @@ module Vendor_nvidia = struct
       - Find the vgpuType elements that match the vgpu type ids found above,
         and construct vgpu_conf records.
     *)
+    let host_driver_version = get_host_driver_version () in
+    let driver_support_multiple_vgpu = is_host_driver_support_multi_vgpu ~host_driver_version ~supported_versions:!Xapi_globs.nvidia_multi_vgpu_enabled_driver_versions in
```

Comment:
Again, grammatically awkward. 
* `driver_supports_multiple_vgpus` 
* `multi_vgpu_support`
Avoid overlong lines in new code.

Reply:
`multi_vgpu_supported` is actually from Nvidia config file `multiVgpuSupported`, I will change the others.